aws/requests.yaml

```diff
@@ -1,4 +1,9 @@
 releases:
+    - name: "> 12.3.0, < 13.0.0"
```

Comment:
So this includes cert manager v2.3.0 in **all** AWS releases between (but not including) 12.3.0 and 13.0.0, right?

Just curious, what happens when there's a new cert manager v2.4.0? Can we override the v2.3.0 that's already released?

Reply:
This means it can be 2.3.0 or newer but it can stay as 2.3.0. If you want to enforce 2.4.0 you edit this request from 2.3.0 to 2.4.0 and all new releases (created after your edit) need to include 2.4.0

Reply:
cool! 😎 thanks!